### DISCUSSION TITLE
Better pasting to Obsidian

### DISCUSSION THREAD

**Discussion by david-ps** [Hugo-COLLIN/SaveMyPhind-conversation-exporter#137] [Ideas]
When I paste to Obsidian, additional blank and horizontal lines are included (see below). 
Remove or provide an option to exclude this formatting? Thanks :)

```
## User
Create a checklist for Inconsistent Progress Tracking

## AI answer
Checklist for Inconsistent Progress Tracking
--------------------------------------------
```

**Comment by Hugo-COLLIN** [comment]
Hello @david-ps,
Thank you for your feedback.

True additional blank could be avoided.

Horizontal lines are part of the exported content and means the line below is interpreted as a title (this syntax is an equivalent to `# Title`). It's true cohabitation of these two syntaxes could be annoying. But it's also a way to distinguish content from the file structure (AI answer containing a `# Title 1` under title 2 `## AI answer` would break the markdown titles hierarchy). 

Maybe exporting AI answers as plain text inside the markdown-structured file is what you need ? Please let me know.

Let users cutomize how the file is formatted sound (complicated but) great!

I'll surely open issue(s) from this conversation.
